src/components/FetchTest.js

```diff
@@ -30,7 +30,7 @@ export default function FetchTest() {
 
   useEffect(()=> {
     fetchData()
-  })
+  }) // dependency array
```

Comment:
this is really bad :P When you dont pass a dependency array, useEffect will re run it's effect every render. you almost definitely only wanted to call this network request once, so you should explicitly pass an empty array as the second parameter.

```js
  useEffect(()=> {
    fetchData()
  }, [])  // <--- note the empty array
```